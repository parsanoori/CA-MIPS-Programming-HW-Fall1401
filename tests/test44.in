54
603070695
1058665487
557896488
732454977
421509001
990624383
628410874
525221885
872809178
527550911
199358299
166797830
282251650
244207425
445109225
816181788
960273857
660377885
726010702
121443271
288600979
1030770423
204464753
100013521
264087172
913948151
220805947
924886842
122714005
294899406
41770592
230879977
515859965
627618745
959642972
63590064
641311594
444507297
638789948
255698864
533494163
647378986
439987775
948322071
980576229
850883171
370847128
44259317
1058367299
659355449
997208700
16639352
647388168
953479055