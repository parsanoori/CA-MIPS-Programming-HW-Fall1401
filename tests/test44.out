16639352
41770592
44259317
63590064
100013521
121443271
122714005
166797830
199358299
204464753
220805947
230879977
244207425
255698864
264087172
282251650
288600979
294899406
370847128
421509001
439987775
444507297
445109225
515859965
525221885
527550911
533494163
557896488
603070695
627618745
628410874
638789948
641311594
647378986
647388168
659355449
660377885
726010702
732454977
816181788
850883171
872809178
913948151
924886842
948322071
953479055
959642972
960273857
980576229
990624383
997208700
1030770423
1058367299
1058665487